test/e2e/workflow/deployment_test.go

```diff
@@ -160,6 +161,50 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			CheckConfigCondition(ConditionAvailable, ConditionTrue, 10*time.Minute, 30*time.Second)
 		})
 	})
+
+	Context("testing range generation after kubeMacPool reinstalation ", func() {
+		var (
+			config                                           opv1alpha1.NetworkAddonsConfigSpec
+			oldRangeEnd, oldRangeStart, rangeEnd, rangeStart string
+		)
+
+		BeforeEach(func() {
+			config = opv1alpha1.NetworkAddonsConfigSpec{KubeMacPool: &opv1alpha1.KubeMacPool{}}
+			CreateConfig(config)
+		})
+
+		AfterEach(func() {
+			DeleteConfig()
```

Comment:
you can drop AfterEach, it's already defined https://github.com/kubevirt/cluster-network-addons-operator/blob/master/test/e2e/workflow/main_test.go#L45